content/post/mirror-help/raspberry-pi-os-images.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "raspberry-pi-os-images"
+tags = ["mirror-help"]
+author = "danyang685"
++++
+
+主仓库地址： http://downloads.raspberrypi.org/
+
+**使用方法**
+
+进入以下包含 Raspbreey Pi OS 系统镜像的目录，选择`.zip`文件下载即可。推荐使用 [balenaEtcher](https://www.balena.io/etcher/) 将镜像写入 SD 卡
+
+- [raspios_lite_armhf](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspios_lite_armhf/images/)：精简版 Raspbreey Pi OS 系统（下载大小约2.3GB）
+- [raspios_armhf](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspios_armhf/images/)：常规版 Raspbreey Pi OS 系统（下载大小约2.3GB）
+- [raspios_full_armhf](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspios_full_armhf/images/)：完整版 Raspbreey Pi OS 系统（下载大小约2.3GB）
+- [raspios_lite_arm64](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspios_lite_arm64/images/)：精简版 Raspbreey Pi OS 64位系统（下载大小约2.3GB）
+- [raspios_arm64](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspios_arm64/images/)：常规版 Raspbreey Pi OS 64位系统（下载大小约2.3GB）
+
+**备注**
+
+- 暂无 Raspbreey Pi OS 64位完整版系统
+- Raspbreey Pi OS 是 Raspbian 的后继者，二者名称不同，但功能和界面相同。首个 Raspbreey Pi OS 系统发行于 2020年5月27日
+- 2012年7月15日-2020年2月14日期间发行的旧版 Raspbian 系统镜像与 Raspbreey Pi OS 位于不同目录，各版本分别位于：[raspbian_lite](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspbian_lite/images/)、[raspbian](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspbian/images/)、[raspbian_full](https://mirrors.sjtug.sjtu.edu.cn/raspberry-pi-os-images/raspbian_full/images/)。
+
+**Raspbreey Pi OS 常用设置**
+
+- 无头初始化设置
+  - 启用 SSH：在 SD 卡 boot 分区中创建名为 `ssh` 或 `ssh.txt` 的空白文件
+
+  - 预设 WiFi 信息：在 SD 卡 boot 分区中创建名为 `wpa_supplicant.conf` 的文本文件，按如下格式填写 `YOUR_WiFi_NAME` 和 `YOUR_WiFi_PASSWORD` ，推荐设置为本机 WiFi 热点或拥有管理员权限的无线路由器 WiFi ，以便查找树莓派的 IP 地址
+
+    ```
+    ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
+    update_config=1
+    country=CN
+    
+    network={
+    	ssid="YOUR_WiFi_NAME"
+    	psk="YOUR_WiFi_PASSWORD"
+    }
+    ```
+
+  - 远程连接：插卡启动树莓派，在 WiFi 热点中查询 IP 分配情况，确认树莓派的 IP 地址，推荐使用 [PuTTY](https://mirrors.sjtug.sjtu.edu.cn/putty/) 进行连接。树莓派的默认 SSH 端口为 22，用户名为 `pi` ，默认密码为 `raspberry` 
+
+- 命令行设置
+
+  - 使用显示屏和键盘，或使用 SSH 远程连接到树莓派，输入 `sudo raspi-config` 命令进行基本设置，包括时区、语言、网络、外设等。
+
+- 有头图形界面设置
+
+  - 首次启动时，根据提示进行鼠标操作设置即可
+
+- apt 软件源修改
+
+  - Raspbreey Pi OS 系统镜像内预置的 apt 软件源在国内访问缓慢，可将 apt 软件源更改为 SJTUG 镜像源
+
+    - 将 `/etc/apt/sources.list` 文件中的 raspbian 存储库由 http://raspbian.raspberrypi.org/raspbian/ 更改为 https://mirrors.sjtug.sjtu.edu.cn/raspbian/raspbian/
+    - 将 `/etc/apt/sources.list.d/raspi.list` 文件中的 raspberrypi 存储库由 http://archive.raspberrypi.org/debian/ 更改为 https://mirrors.sjtug.sjtu.edu.cn/raspberrypi/debian/
+
+  - 可输入以下命令完成 apt 软件源设置
+
+    ```shell
+    sudo cp /etc/apt/sources.list /etc/apt/sources.list.bak
+    sudo cp /etc/apt/sources.list.d/raspi.list /etc/apt/sources.list.d/raspi.list.bak
+    sudo sed -i "s?http://raspbian.raspberrypi.org/raspbian/?https://mirrors.sjtug.sjtu.edu.cn/raspbian/raspbian/?g" /etc/apt/sources.list
```

Comment:
Would you please verify if this command works?

Reply:
OK!